### PR TITLE
ArtifactEntity.tags in ArtifactEntryRepositoryTest.crudShouldWork() Fix

### PR DESCRIPTION
**Pull request by anki2189**
This PR closes strongbox/strongbox#1645

### CODE REVIEW

File: src/test/java/org/carlspring/strongbox/janusgraph/reposiotries/ArtifactEntryRepositoryTest.java [hunk 1]
```diff
@@ -92,10 +92,9 @@ public void crudShouldWork()
         assertEquals(createdOn, artifactEntrySaved.getCreated());
         Set<String> tags = artifactEntrySaved.getTags();
         assertNotNull(tags);
-        //TODO: fix tags
-        //assertEquals(2, tags.size());
-        //assertTrue(tags.contains("release"));
-        //assertTrue(tags.contains("stabile"));
+        assertEquals(2, tags.size());
+        assertTrue(tags.contains("release"));
+        assertTrue(tags.contains("stabile"));
```
Comment (carlspring) [inline src/test/java/org/carlspring/strongbox/janusgraph/reposiotries/ArtifactEntryRepositoryTest.java]: Cosmetic: There's a typo. It should be `stable`.

Comment (anki2189) [reply src/test/java/org/carlspring/strongbox/janusgraph/reposiotries/ArtifactEntryRepositoryTest.java]: Fixed

**Comment by sbespalov** [comment]
@anki2189 thanks for interest to this issue. Let me provide some comments on it. The thing is that the 
`SetPropertyConverter` is not what whe need here. We should store `Vertex` property with `org.janusgraph.core.Cardinality.SET` instead of serializing it to single string.

**Comment by carlspring** [comment]
@sbespalov ,

What's the status of this task? Do you have any further hints for @anki2189 , or is this task `on hold`?

**Comment by sbespalov** [comment]
@carlspring I'm not working on this and it still relevant.

**Comment by sbespalov** [comment]
the current problem seems related with the way that cypher-for-gremlin save the data
because I found that it save the collection property values as string
like "[one, two, three]"
as an option we can try to inline collection values into Cypher query, this should be done in `GremlinRequest.executeRequest(Statement query)`

**Comment by carlspring** [comment]
Okay... So, I assume we need to have a separate task for that to be done generically, the code for which can then be used for completing this task? Am I getting this right? If so, could you please create a new task for it and link it as a dependency of this one? Thanks!

**Comment by sbespalov** [comment]
I'm not sure that this should be a separate task. How many tasks do we need to solve one problem? :)

**Comment by carlspring** [comment]
I thought it might be possible to do this handling in a more generic way in a base classs. Apparently, I must have misunderstood.

**Comment by sbespalov** [comment]
@carlspring you understand right, I've mentioned in task description that it should be fixed in `GremlinRequest`, which is generic class.

**Comment by carlspring** [comment]
I see what you mean, thanks for clarifying!

**Comment by sbespalov** [comment]
this seems no longer relevant, the `ArtifactEntity.filenames` property seems works fine in [Migrate Strongbox from OrientDB to JanusGraph #1676](https://github.com/strongbox/strongbox/pull/1676).

**Pull request closed without merging**